kalite/settings/base.py

```diff
@@ -466,6 +466,9 @@
 SESSION_COOKIE_AGE = 60 * 30     # 30 minutes
 SESSION_SAVE_EVERY_REQUEST = True
 
+# Expire session cookies whenever we close the browser.
```

Comment:
I believe this is intentionally removed, as it does not work as a setting (seems to fail on most browsers).